db/seeds.rb

```diff
@@ -71,10 +71,15 @@
 
   puts 'Create Incidental Types'
   incidentals = YAML::load_file(File.join(Rails.root, 'db/db_yml', 'incidental_types.yml'))
-  incidentals.each do |incidental|
-    IncidentalType.where(name: incidental['name']).first_or_create incidental
+  incidentals.each do |i|
+    IncidentalType.where(name: i['name'],
+                         description: i['description'],
+                         base: i['base'],
+                         modifier_amount: i['modifier_amount'],
+                         modifier_description: i['modifier_description']).first_or_create
```

Comment:
Any chance that you can just use the local variable, as we do [here](https://github.com/anmerkl/probable-engine/blob/7769d673f673e5c18a3f647a0b677ab9cc21ecfd/db/seeds.rb#L47)? That would turn this into...

```Ruby
incidentals.each do |incidental|
  IncidentalType.where(incidental).first_for_create
end
```